Directory.Packages.props

```diff
@@ -37,8 +37,8 @@
             1. Verifying with the core Azure Functions host team that there will not be any issues
             2. Running the bundle dependency test and verifying it passes
      -->
-
-    <PackageVersion Include="Azure.Identity" Version="1.10.2"/>
+    <PackageVersion Include="Azure.Core" Version="1.41.0"/> <!-- Updated to address security issue, should be removed once Azure.Identity is updated-->
```

Comment:
Why are we adding a new library to address security issue? Will it be okay if we remove it after Azure.Identity is updated?

Reply:
We are not adding it (it was already brought in as a transitive dependency)

This is just ensuring that we get the correct version with the fix for the issue, since the transitive dependency brings in an older version.